_aggregations/metric/average.md

```diff
@@ -28,7 +39,9 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
 ```
 {% include copy-curl.html %}
 
-#### Example response
+## Example response
+
+The response is as follows:
```

Comment:
Could you make the intro sentence more descriptive? Something like "The response contains...."